### ISSUE TITLE
Text inside a <code> HTML element Is Not Readable In Google Chrome

### ISSUE THREAD

**Issue by a2hsh** [nvaccess/nvda#11111]
### Issue Description:
When encountering a code element inside another interactive element in Google Chrome, NVDA skips reading what is inside the code element.
This issue accured in Google chrome, it did not accure in Firefox.

### Steps to reproduce:
In Google Chrome, Go to any web page that has a code element inside another interactive element -A link for example-, [This guide on SQLAlchemy would suffice.](https://flask-sqlalchemy.palletsprojects.com/en/2.x/models/ "Visit an example page in which the issue accures")

### Actual behavior:
NVDA does not read the text inside the code element, but rather skips it intirely.

### Expected behavior:
NVDA should read the text inside the code element, as it's visible in the developer tools in Google Chrome.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2019.3.1

#### Windows version:
Windows 10 Version 1903 (OS Build 18362.778)

#### Name and version of other software in use when reproducing the issue:
Google Chrome, Version 81.0.4044.129 (Official Build) (64-bit)

#### Does the issue still occur after restarting your computer?
Yes, the issue still persists after restarting.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
this issue is present in NVDA 2019.1, as well as the 2020.1 RC1 releases.

#### If addons are disabled, is your problem still occuring?
Yes.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes.

### Supporting Files:
I uploaded a zip file which has my NVDA log, as well as an HTML webpage I created to illustrate the issue:
[LogAndExample.zip](https://github.com/nvaccess/nvda/files/4578716/LogAndExample.zip)

**Comment by leonardder** [comment]
I tried this in the following browsers:

- MIcrosoft Edge stable (81)
- MIcrosoft Edge Canary (84)
- Google Chrome Canary (84)

It looks like in the latter two, this issue is fixed.

**Comment by Adriani90** [comment]
Chrome 84 is stable now, I am closing this as works for me.
